src/api/public/apidocs/paths/source_project_name_package_name_cmd_branch.yaml

```diff
@@ -76,6 +76,18 @@ post:
         enum:
           - 1
       description: Set to ignore validation and resolving of devel package or devel project definition.
+    - in: query
+      name: force
+      schema:
+        type: string
+        enum:
+          - 1
+      description: Set to allow overwrite of a pre-existing package.
```

Comment:
I would use `allow overwriting` or `allow to overwrite`. See this [link](https://www.grammaring.com/to-infinitive-or-gerund-advise-recommend-allow-permit-forbid-require).

Reply:
Updated